frontend/tests/login.test.js

```diff
@@ -1,7 +1,7 @@
 import { expect, test } from '@playwright/test';
 
 test('Page has header and message', async ({ page }) => {
-	await page.goto('http://localhost:5173/');
```

Comment:
Test should only test `localhost` as it will work for development and CI purposes. Testing the actual domain is redundant, as the local test should have been done before merging.